### ISSUE TITLE
automated bitcoin->amazon gift card

### ISSUE THREAD

**Issue by papcoin** [bitpay/copay#4704]
hi,

is there anyway we can integrate the amazon GC with our web system through copay?

so deposit on copay->then buy amazon gc from copay through api or something

thanks !

**Comment by matiu** [comment]
Could you explain more in detail how the integration would be so we can provide you an accurate answer?  Who will be purchasing the GC?

**Comment by papcoin** [comment]
1. it will be our company

2. the problem with current system is , we have about 40 employee, it means if everyone wants to buy amazon they have to install copay 40 times in ther individual laptop, not mentioning must top up 40 times individual account

3. to overcome this, we plan to make integration , so that we have master balance in copay and everytime needs GC, we can just cut it from master balance , and give The GC automatically to employee via our website employee area / sms, etc


in short explanation, we need API like json/ else from copay to purchase the GC 

possible? 
thankS !!

**Comment by papcoin** [comment]
bumppp

**Comment by matiu** [comment]
no, I dont thing that is possible for me moment. We will let you know any change regarding this.
